src/main/java/net/sf/jabref/logic/formatter/bibtexfields/NormalizeDateFormatter.java

```diff
@@ -59,7 +60,7 @@ public String getDescription() {
                 "uuuu-M-d", "uuuu-M",
                 "M/uu", "M/uuuu",
                 "MMMM d, uuuu", "MMMM, uuuu",
-                "d.M.uuuu"};
+                "d.M.uuuu", "uuuu.M.d"};
```

Comment:
Almost all my .bib-files have that format (or rather uuuu.MM.dd, but this seems to cover that as well). Not sure why, but the files are created on both Linux and Windows and some are from my (non-Swedish) PhD students, so for some reason it seems quite common. Default in older versions of JabRef? (I would never select that format myself, so I haven't configured it for sure.)